target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -186,6 +186,13 @@ zbtlink,zbt-we1226)
 	ucidef_set_led_switch "lan2" "LAN2" "green:lan2" "switch0" "0x02"
 	ucidef_set_led_switch "wan" "WAN" "green:wan" "switch0" "0x10"
 	;;
+zbtlink,zbt-we2426-b)
+	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x10"
+	ucidef_set_led_switch "lan1" "lan1" "green:lan1" "switch0" "0x01"
+	ucidef_set_led_switch "lan2" "lan2" "green:lan2" "switch0" "0x02"
+	ucidef_set_led_switch "lan3" "lan3" "green:lan3" "switch0" "0x04"
+	ucidef_set_led_switch "lan4" "lan4" "green:lan4" "switch0" "0x08"
```

Comment:
```suggestion
	ucidef_set_led_switch "lan1" "lan1" "green:lan-1" "switch0" "0x01"
	ucidef_set_led_switch "lan2" "lan2" "green:lan-2" "switch0" "0x02"
	ucidef_set_led_switch "lan3" "lan3" "green:lan-3" "switch0" "0x04"
	ucidef_set_led_switch "lan4" "lan4" "green:lan-4" "switch0" "0x08"
```
IIRC `function-enumerator` adds a dash, so these are the correct labels.

Reply:
Dashes added. Thank you for help.